cycode/cli/files_collector/iac/tf_content_generator.py

```diff
@@ -60,8 +68,10 @@ def _extract_resources(tfplan: str, filename: str) -> List[ResourceChange]:
         for resource_change in resource_changes:
             resources.append(
                 ResourceChange(
+                    module_address=resource_change.get('module_address'),
```

Comment:
we are using `.get()` which will return `None` if the field does not exist. do we expect this? if so, should we mark these fields as `Optional[]` in `ResourceChange` model? Other fields that we had before do not used with `.get()`

Reply:
Yes those are optional. Fixed.